### PR TITLE
Add requires headers in plugin

### PR DESCRIPTION
**Pull request by spacedmonkey**
For reference, it seems this was added in WordPress 5.4 here:
https://core.trac.wordpress.org/ticket/46938

### CODE REVIEW

**Comment by peterfabian** [comment]
Hi, thanks for raising this! 

I'm just a bit confused, it seems the patch has not been merged yet to WP, has it? Is it planned for 5.5? As far as I can see, the header expects a different string?

https://github.com/WordPress/WordPress/blob/38b7495af752f223b519fe423a71513eaca74591/wp-admin/includes/plugin.php#L80

**Comment by claudiosanches** [comment]
@peterfabian 

> I'm just a bit confused, it seems the patch has not been merged yet to WP, has it? Is it planned for 5.5?

According to https://core.trac.wordpress.org/ticket/46938 it got introduced in 5.3.

File: woocommerce.php
Comment (claudiosanches) [inline woocommerce.php:11]: This need to be updated to:

```
* Requires at least: 5.2
```

Comment (claudiosanches) [inline woocommerce.php:11]: Please change this one to:

```
* Requires at least: 5.2
```

**Review by claudiosanches — DISMISSED** [review]
Thank you for your help, but there's a small change to be done before this could be approved.

According with the docblock of [`get_plugin_data()`](https://github.com/WordPress/WordPress/blob/38b7495af752f223b519fe423a71513eaca74591/wp-admin/includes/plugin.php#L46) function it should use "Requires at least" instead of "Requires WP".

**Comment by peterfabian** [comment]
As far as I can see, the code in WP master and 5.4 uses 'Requires at least' and not 'Requires WP', so that's what I was ultimately trying to say...

**Comment by spacedmonkey** [comment]
My bad, I put the wrong header, as I looked at the patch and not the [commit](https://core.trac.wordpress.org/changeset/45546). These headers were added in 5.3 by the site health team. I know, I am a maintainer of site health in core.

File: woocommerce.php
Comment (spacedmonkey) [reply woocommerce.php]: Fixed.

**Review by peterfabian — APPROVED** [review]
LGTM, thanks!

**Pull request merged**
